pkg/network/protocols/http2/telemetry.go

```diff
@@ -57,15 +57,15 @@ func newHTTP2KernelTelemetry() *kernelTelemetry {
 
 // update updates the kernel metrics with the given telemetry.
 func (t *kernelTelemetry) update(tel *HTTP2Telemetry) {
-	t.http2requests.Add(int64(tel.Request_seen))
-	t.http2responses.Add(int64(tel.Response_seen))
-	t.endOfStream.Add(int64(tel.End_of_stream))
-	t.endOfStreamRST.Add(int64(tel.End_of_stream_rst))
-	t.pathExceedsFrame.Add(int64(tel.Path_exceeds_frame))
-	t.exceedingMaxInterestingFrames.Add(int64(tel.Exceeding_max_interesting_frames))
-	t.exceedingMaxFramesToFilter.Add(int64(tel.Exceeding_max_frames_to_filter))
+	t.http2requests.UpdateCounterWithDifference(int64(tel.Request_seen))
+	t.http2responses.UpdateCounterWithDifference(int64(tel.Response_seen))
+	t.endOfStream.UpdateCounterWithDifference(int64(tel.End_of_stream))
+	t.endOfStreamRST.UpdateCounterWithDifference(int64(tel.End_of_stream_rst))
+	t.pathExceedsFrame.UpdateCounterWithDifference(int64(tel.Path_exceeds_frame))
+	t.exceedingMaxInterestingFrames.UpdateCounterWithDifference(int64(tel.Exceeding_max_interesting_frames))
+	t.exceedingMaxFramesToFilter.UpdateCounterWithDifference(int64(tel.Exceeding_max_frames_to_filter))
 	for bucketIndex := range t.pathSizeBucket {
-		t.pathSizeBucket[bucketIndex].Add(int64(tel.Path_size_bucket[bucketIndex]))
```

Comment:
That "breaks" the dashboards as we use monotonic difference in them
If a counter is not relevant, we might use Gauge value instead and use `Set` methods

Reply:
I'm uncertain why it breaks the dashboard, everything should still be monotonically increasing. Regardless, I'm now using Gauge